tools/internal_ci/helper_scripts/prepare_build_linux_perf_rc

```diff
@@ -19,9 +19,10 @@
 ulimit -n 32768
 ulimit -c unlimited
 
-sudo pip install tabulate
+python3 -m pip install --upgrade pip
```

Comment:
nit: let's use a pinned version of pip `19.3.1`. Bad release of pip broke us before.

Reply:
Good catch, thanks. I wasn't aware of that.